CODE-OF-CONDUCT.md

```diff
@@ -38,10 +42,11 @@ Examples of unacceptable behavior include:
 
 ## Enforcement Responsibilities
 
-Community leaders are responsible for clarifying and enforcing our standards of
-acceptable behavior and will take appropriate and fair corrective action in
-response to any behavior that they deem inappropriate, threatening, offensive,
-or harmful.
+Community leaders (including maintainers, Working Group leads, the Technical
```

Comment:
I would update this to “Members of steering committee” are responsible...

Reply:
+1 - this is in line with our discussion that Steering should be responsible for "taking appropriate and fair corrective action" and not putting the burden on e.g. individual WG leads.

Reply:
I think his has now been addressed in the text of the doc: `(including maintainers, Working Group leads, the Technical Oversight Committee and the Steering Committee)`